tools/serverpod_cli/test/generator/dart/server_code_generator/database_repository_class_list_relation_additions_test.dart

```diff
@@ -171,13 +171,12 @@ void main() {
         expect(peopleMethod, isNotNull, reason: 'Missing people method.');
       });
 
-      test('people method has the input params of session, example and person',
+      test(
+          'people method has the input params of session, example, person and named param transaction',
           () {
         expect(
           peopleMethod?.parameters?.toSource(),
-          matches(
-            r'(_i\d.Session session, Example example, Person person)',
-          ),
+          r'(_i1.Session session, Example example, Person person, {_i1.Transaction? transaction})',
```

Comment:
Suggestion: Revert back to `\d` since the order of import does not matter.

Reply:
Same for the other tests.

Reply:
Brought back regex matching and fixed them: [282b62e](https://github.com/serverpod/serverpod/pull/2656/commits/282b62e86e6491db117e5028e0a985c6672f0429)